todoApp/src/components/DisplayTodo.tsx

```diff
@@ -1,28 +1,43 @@
-//Stateful component
 import React, { useState } from "react";
 import { FiCheckCircle } from "react-icons/fi"; //React icons
 import styles from "./DisplayTodo.module.css";
+import { TodoListInterface } from "../App";
 
-interface TodoListsInterface {
+interface DisplayTodoPropInterface {
+  id: string;
   title: string;
+  isCompleted: boolean;
 }
 
-const DisplayTodo = (props: TodoListsInterface) => {
-  const [todoTask, setTodoTask] = useState(true);
+const DisplayTodo = ({ id, title, isCompleted }: DisplayTodoPropInterface) => {
+  const [todoCompleted, setTodoCompleted] = useState(isCompleted);
 
-  const changeStatusHandler = () => {
-    setTodoTask((prevState) => !prevState);
+  const getHandler = (id: string) => {
```

Comment:
This approach of marking todo as done will lead to **state out of sync** error.
1. Add a todo item
2. Check todo array state (isCompleted field) inside React devtools for added todo.
3. Mark todo as completed.
4. Check todo array state again for isCompleted field and observe the result.